packages/edit-site/src/components/dataviews/README.md

```diff
@@ -47,7 +47,6 @@ Example:
 		{ field: 'author', operator: 'in', value: 2 },
 		{ field: 'status', operator: 'in', value: 'publish,draft' }
 	],
-	visibleFilters: [ 'author', 'status' ],
```

Comment:
We no longer need this, the visible filters are derived from the `filter` state.